src/views/CollectiveView.vue

```diff
@@ -39,12 +41,38 @@ export default {
 		...mapGetters([
 			'currentCollective',
 			'loading',
+			'printLink',
 			'showing',
 		]),
 	},
 
+	mounted() {
+		window.addEventListener('keydown', this.printKeyHandler)
+	},
+
+	beforeDestroy() {
+		window.removeEventListener('keydown', this.printKeyHandler)
+	},
+
 	methods: {
 		...mapMutations(['hide']),
+
+		/**
+		 * @param {KeyboardEvent} event the keydown event
+		 */
+		printKeyHandler(event) {
+			// Handle `CTRL+P` or `CMD+P` but ensure ALT or SHIFT are NOT pressed (e.g. CTRL+SHIFT+P is new private tab on firefox)
+			if ((event.metaKey || event.ctrlKey) && event.key.toLowerCase() === 'p' && !(event.altKey || event.shiftKey)) {
+				const handle = window.open(this.printLink, 'ncCollectivesPrint').focus()
```

Comment:
Unfortunately, this doesn't work for me with Firefox: `Uncaught TypeError: window.open(...) is null`

Reply:
This happens if you have a popup blocker (see `if`-`else` below)...
The `.focus()` is required to go into the `else` part (only execute it if a handle is given) :see_no_evil:

Fixed this.